### PR TITLE
AnimatedSprite warnings should now include the path and animation nam…

### PR DESCRIPTION
**Pull request by rcurtis**
…e that generated the warning.

### CODE REVIEW

File: scene/2d/animated_sprite.cpp [hunk 1]
```diff
@@ -355,23 +355,32 @@ void AnimatedSprite::_notification(int p_what) {
 		case NOTIFICATION_DRAW: {
 
 			if (frames.is_null()) {
-				print_line("no draw no faemos");
+				print_line("no draw no faemos: " + String(get_path()));
```
Comment (HummusSamurai) [inline scene/2d/animated_sprite.cpp]: What is "faemos"?

File: scene/2d/animated_sprite.cpp [hunk 2]
```diff
@@ -355,23 +355,32 @@ void AnimatedSprite::_notification(int p_what) {
...
 				return;
 			}
 
 			if (frame < 0) {
-				print_line("no draw frame <0");
+				print_line("no draw frame <0: " + String(get_path()));
```
Comment (HummusSamurai) [inline scene/2d/animated_sprite.cpp]: This also is not explicit.

File: scene/2d/animated_sprite.cpp (hunk 1 above)
Comment (rcurtis) [reply scene/2d/animated_sprite.cpp]: I have no idea what "faemos" is, I assume is Portuguese, but it is what was already there.

File: scene/2d/animated_sprite.cpp (hunk 2 above)
Comment (rcurtis) [reply scene/2d/animated_sprite.cpp]: Its not explicit, but it is the phrasing that was already being used.

Comment (HummusSamurai) [reply scene/2d/animated_sprite.cpp]: Sure, not implying it's your fault, just pointing it out so other reviewers can see it.

Comment (rcurtis) [reply scene/2d/animated_sprite.cpp]: I understand, just wanted to give my justification for not changing it.  I don't know if someone else really needs that phrasing.

**Comment by Noshyaar** [comment]
I'd suggest using `ERR_FAIL_COND` and `ERR_EXPLAIN` macros instead.

**Comment by akien-mga** [comment]
Those `print_line` are debug info added by the developer, not warnings meant for the end user. They should just be removed, not improved.

**Pull request closed without merging**